tests/acceptance/conftest.py

```diff
@@ -0,0 +1,108 @@
+from contextlib import contextmanager
+
+import pytest
+
+from databuilder.backends import TPPBackend
+from databuilder.dsl import Cohort
+from tests.lib.tpp_schema import (
+    apcs,
+    ctv3_event,
+    negative_test,
+    organisation,
+    patient,
+    patient_address,
+    positive_test,
+    registration,
+    snomed_event,
+)
+from tests.lib.util import extract
+
+
+@pytest.fixture(scope="module")
```

Comment:
This module scope is a mistake on my part. There was an older version of this where I had split each variable definition into its own test function and I needed this to avoid re-populating the database every time because that would be too slow. But now that each of these study tests consists of just a single function this is no longer necessary.

Reply:
Thanks for the comment. I've removed the scope, so now the fixture is function-scoped (the default).